clients/benchmarks/CMakeLists.txt

```diff
@@ -91,4 +91,22 @@ endif( )
 set_target_properties( rocblas-bench PROPERTIES CXX_EXTENSIONS NO )
 set_target_properties( rocblas-bench PROPERTIES RUNTIME_OUTPUT_DIRECTORY "${PROJECT_BINARY_DIR}/staging" )
 
+set( ROCBLAS_DGEMM_PERF_YAML "${CMAKE_CURRENT_BINARY_DIR}/dgemm_perf.yaml")
```

Comment:
This builds the dgemm_perf.yaml file from a Python script which is much more compressed than the YAML.
It then builds a binary file from the YAML file, for direct input to rocblas-bench.